src/classes/VOL_CTRL_PersonalSiteContactInfo_TEST.cls

```diff
@@ -30,61 +30,56 @@
 
 @isTest
 private with sharing class VOL_CTRL_PersonalSiteContactInfo_TEST {
-
+    private static VOL_Access_TEST.Stub accessMock = new VOL_Access_TEST.Stub();
+    
     //******************************************************************************************************
     // Test Code
 
+    private static void setAccessMock() {
+        VOL_CTRL_PersonalSiteContactInfo.access = (VOL_Access) Test.createStub(VOL_Access.class, accessMock);
+    }    
+    
+    private static testMethod void shouldCheckAccessOnSave(){
```

Comment:
Can we use the IsTest annotation here and add a space before the opening curly brace?